modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/BaseSourceCheck.java

```diff
@@ -647,13 +665,31 @@ protected String getVariableTypeName(
 			return variableTypeName;
 		}
 
-		return _getVariableTypeName(
-			fileContent, variableName, includeArrayOrCollectionTypes);
+		JavaClass javaClass = JavaClassParser.parseJavaClass(fileName, fileContent);
+
+		for (JavaTerm childJavaTerm : javaClass.getChildJavaTerms()){
+			if (childJavaTerm.isJavaVariable()){
+
```

Comment:
@TamyrisTorres, please run SF in this commit. We shouldn't be adding more commits just to fix very simple SF issues. :wink: